src/msg/async/rdma/Infiniband.h

```diff
@@ -486,6 +497,8 @@ class Infiniband {
     uint32_t     max_recv_wr;
     uint32_t     q_key;
     bool dead;
+    std::atomic<uint32_t> tx_wr; // atomic counter for successful Tx WQEs
```

Comment:
better off initialize it like:
```c++
std::atomic<uint32_t> tx_wr{0};
```
or 
```c++
std::atomic<uint32_t> tx_wr = {0};
```
s/atomic counter/counter/

Reply:
OK, and with such initialization we should also get rid of the following lines in Infiniband::QueuePair::QueuePair():
+  tx_wr(0),
+  tx_wc(0)